src/test/fuzz/kitchen_sink.cpp

```diff
@@ -23,4 +34,21 @@ void test_one_input(const std::vector<uint8_t>& buffer)
     (void)JSONRPCTransactionError(transaction_error);
     (void)RPCErrorFromTransactionError(transaction_error);
     (void)TransactionErrorString(transaction_error);
+
+    (void)StringForFeeEstimateHorizon(fuzzed_data_provider.PickValueInArray({FeeEstimateHorizon::SHORT_HALFLIFE, FeeEstimateHorizon::MED_HALFLIFE, FeeEstimateHorizon::LONG_HALFLIFE}));
+
+    const OutputType output_type = fuzzed_data_provider.PickValueInArray({OutputType::LEGACY, OutputType::P2SH_SEGWIT, OutputType::BECH32});
+    const std::string& output_type_string = FormatOutputType(output_type);
+    OutputType output_type_parsed;
+    bool parsed = ParseOutputType(output_type_string, output_type_parsed);
+    assert(parsed);
+    assert(output_type == output_type_parsed);
+    (void)ParseOutputType(fuzzed_data_provider.ConsumeRandomLengthString(64), output_type_parsed);
+
+    CAddrMan addr_man;
+    CDataStream data_stream_peers = ConsumeDataStream(fuzzed_data_provider);
```

Comment:
Same here. Creating an addrman stream seed by hand requires a random length string first.

Reply:
Thanks for reviewing. Now skipping all changes that are unrelated to CAddrMan to keep this PR focused.